client/src/js/app.js

```diff
@@ -746,11 +746,17 @@
     });
 
     $translateProvider.useSanitizeValueStrategy('escape');
-
-    //TODO Try and assign the previous sessions language key here
+  
     $translateProvider.preferredLanguage('fr');
   }
 
+  function localeConfig(tmhDynamicLocaleProvider) { 
+
+    // TODO Hardcoded default translation/ localisation
+    tmhDynamicLocaleProvider.localeLocationPattern('/i18n/locale/angular-locale_{{locale}}.js');
+    tmhDynamicLocaleProvider.defaultLocale('fr-fr');
```

Comment:
No problem with that. This does show a fairly lazy thing we do here using 'fr'/ 'fr-fr' strings as default.
This doesn't work if you don't have French for example. It's not the end of the world because it should just default back to angular's default (en-US) if it fails but the feature is clearly useful. 

My first naive thoughts are that you could template into the file/ index.html the enterprises default language and locale. This way it is ready to use during the angular bootstrap process. (app.js). (process mentioned brielfy here #663 )